.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Aexpect CI
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  selftests:
+    name: ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.x]
```

Comment:
I'd make last version explicit and write directly 3.9. 
Travis was testing in 3.6, 3.7 and 3.8. Is there any reason to skip 3.6 and 3.7 here?

Reply:
We always want to cover the latest python so I'd prefer to follow the latest one of the supported pythons. As for the matrix I don't want to unnecessary abuse the github free workers. Covering old(ish) and latest python should catch most of the issues.

Reply:
btw, Ana, thank you for your example ci.yml, I simply copied it and adjusted :-)

Reply:
@ldoktor you're welcome!
The tests are very small so I don't think it'll be abuse of builders. And if all these python versions are supported it'd be a good idea to test all of them. 
About 3.x, in some months 3.10 will be released and the actions will switch automatically, this might trigger some false regressions...

Reply:
I like the idea of checking it in 3.7, you never know in which distributions people are running aexpect on.